File: Lang52a.c
#include <stdio.h>

int main() {
    int length;
    int width;
    printf("Enter length: ");
    scanf("%d", &length);
    printf("Enter width: ");
    scanf("%d", &width);
    int area = length * width;
    int perim = 2 * length + 2 * width;
    printf("The area is: %d", area);
    printf("\nThe perimeter is: %d", perim);

    return 0;
}

File: Lang52a.go
package main
import ("fmt")

func main() {
	var length int
	var width int

	fmt.Print("Enter length: ")
	fmt.Scan(&length)
	fmt.Print("Enter width: ")
	fmt.Scan(&width)

	var area int = length * width;
	var perim int = 2 * length + 2 * width

	fmt.Printf("The area is: %d", area)
	fmt.Printf("\nThe perimeter is: %d", perim)
	fmt.Println("\n")
}
